node/network/availability-distribution/src/lib.rs

```diff
@@ -0,0 +1,1827 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! The availability distribution
+//!
+//! Transforms `AvailableData` into erasure chunks, which are distributed to peers
+//! which are intereseted in the relevant candidates.
```

Comment:
```suggestion
//! who are interested in the relevant candidates.
```

Reply:
Never sure what is a "person" and what is a "thing" :>

Reply:
I think both are correct in this case, just to avoid repetition.